CMakeLists.txt

```diff
@@ -528,6 +534,16 @@ if (PROTOBUF)
     )
 endif()
 
+if (USE_SYSTEM_QTSINGLEAPPLICATION)
+    find_path(QTSINGLEAPPLICATION_INCLUDE_DIRS qtsingleapplication.h PATH_SUFFIXES qt${QT_VERSION_MAJOR}/QtSolutions REQUIRED)
+    find_library(QTSINGLEAPPLICATION_LIBRARIES Qt${QT_VERSION_MAJOR}Solutions_SingleApplication-2.6 REQUIRED)
+    set(QTSINGLEAPPLICATION_INCLUDE_DIRS ${QtSingleApplication_INCLUDE_DIRS})
+    set(QTSINGLEAPPLICATION_LIBRARIES ${QtSingleApplication_LIBRARIES})
+else()
+    set(QTSINGLEAPPLICATION_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/qtsingleapplication-2.6_1-opensource/src)
+    set(QTSINGLEAPPLICATION_LIBRARIES "QtSingleApplication")
```

Comment:
This line needs to be removed without replacement. (The bundled QtSingleApplication is added directly to the list of sources, not built as a library target, so there is nothing to link in this case. So `QTSINGLEAPPLICATION_LIBRARIES` should just be left unset.)